library/swift/src/filters/ResponseFilter.swift

```diff
@@ -1,7 +1,28 @@
+@_implementationOnly import EnvoyEngine
 import Foundation
 
+extension EnvoyHTTPFilter {
+  /// Initializer
+  convenience init(responseFilter: ResponseFilter) {
+    self.init()
+    self.name = responseFilter.name
+    self.onResponseHeaders = {
+      let result = responseFilter.onResponseHeaders(ResponseHeaders(headers: $0), endStream: $1)
+      switch result {
+      case .continue(let headers):
+        return [0, headers.headers]
+      case .stopIteration(let headers):
+        return [1, headers.headers]
```

Comment:
Can we name these instead of using 0/1?

Reply:
Yeah, we should. We don't have a great solution from enums across the bridge layer yet, and I had this in place while I thought about it more. But we should at least make them some kind of named constant.

Reply:
Okay, I came up with a solution, which is "don't use enums." Using enums, we'd have to declare and map this set of options 4 times over: first, in Envoy, next at the bridge layer (because Envoy's type is c++ and can't be directly exposed), next in objective-c (because the EnvoyEngine header can't have includes due to the clang issue), next in Swift (because the objective-c type doesn't support associated values). We'd also have to create mapping functions between all 4 variants.

`extern  const`s can be declared in terms of Envoy's implicit internal values and eliminate all but one mapping.

Reply:
(updated)